linera-service/tests/end_to_end_tests.rs

```diff
@@ -8,10 +8,28 @@ mod common;
 
 use common::INTEGRATION_TEST_GUARD;
 use linera_base::identifiers::ChainId;
-use linera_service::cli_wrappers::{Database, LocalNetwork, Network};
+use linera_service::{
+    cli_wrappers::{Database, LocalNetwork, Network},
+    util,
+};
 use linera_views::common::get_table_name;
 use std::time::Duration;
 
+#[tokio::test]
+async fn test_resolve_binary() {
```

Comment:
Doesn't this mean that the tests will fail if the binaries have not been installed beforehand? Is there some invariant which means this is always the case?

Reply:
Got it.